scripts/ci/cmake-v2/ci-ubuntu20.04-gcc11-mpi.cmake

```diff
@@ -28,6 +35,7 @@ MPIEXEC_EXTRA_FLAGS:STRING=--allow-run-as-root --oversubscribe
 MPIEXEC_MAX_NUMPROCS:STRING=${N2CPUS}
 ")
 
+set(CTEST_TEST_ARGS PARALLEL_LEVEL 1)
```

Comment:
Is this needed?

Reply:
I'm not sure whether it's needed.  It was present in about a third of the initial cache files in `scripts/ci/cmake-v2` before this, but I can't see any rhyme or reason to it.  Maybe some builds just behaved in a flaky manner without that limitation.  Should I try to just remove it and see?

Reply:
I see, yes there has been certain level of tuning/tweaking with those params. Maybe remove and see is best here